### PR TITLE
Update activ admin front panel

### PR DESCRIPTION
**Pull request by ChernikovVolodymyr**
Update seeds
Update rspec
Add default values for accommodation_type and transport

### CODE REVIEW

File: app/admin/users.rb [hunk 1]
```diff
@@ -112,12 +112,18 @@ def update_resource(object, attributes)
...
       f.input :comment
       f.input :status_id, as: :select, collection: UserStatus.all.pluck(:status, :id)
-      f.input :coordinator_id, as: :select, collection: AdminUser.all.pluck(:email, :id)
-      f.input :accommodation_pref, as: :select, collection: AccommodationType.all.pluck(:name, :id)
+      f.input :coordinator_id, as: :select, collection: AdminUser.all.pluck(:name, :id)
+      f.input(
+        :accommodation_pref,
+        as: :select,
+        collection: AccommodationType.all.pluck(:name, :id),
+        selected: AccommodationType.all.pluck(:name, :id).first
```
Comment (oleksandr-kychun) [inline app/admin/users.rb]: on edit page selected should use value picked during creation

File: app/admin/users.rb [hunk 2]
```diff
@@ -112,12 +112,18 @@ def update_resource(object, attributes)
...
+      )
       f.input(
         :transport_id,
         as: :select,
-        collection: Transport.to_select_collection
+        collection: Transport.to_select_collection,
+        selected: Transport.to_select_collection.second
```
Comment (oleksandr-kychun) [inline app/admin/users.rb]: on edit page selected should use value picked during creation

File: config/webpacker.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Note: You must restart bin/webpack-dev-server for changes to take effect
 
 default: &default
-  source_path: app/frontend
+  source_path: app/javascript
```
Comment (oleksandr-kychun) [inline config/webpacker.yml]: careful with this, most js and css files are in 'frontend' directory

Comment (ChernikovVolodymyr) [reply config/webpacker.yml]: Ok. It was automaticly

File: package-lock.json [hunk 4]
```diff
@@ -0,0 +1,9103 @@
+{
```
Comment (oleksandr-kychun) [inline package-lock.json]: we already have yarn.lock
do we need this file?

Comment (ChernikovVolodymyr) [reply package-lock.json]: Deleted this file
Addressed here https://github.com/vova/war_shelter/pull/68/commits/3484f23daf598a9867e0898ffe9c8b0683c6ceba

File: config/webpacker.yml (hunk 3 above)
Comment (ChernikovVolodymyr) [reply config/webpacker.yml]: Revirsed
Addressed here https://github.com/vova/war_shelter/pull/68/commits/4c7d826e300e3dfae4bbdd203fdaf45012028228

File: app/admin/users.rb (hunk 1 above)
Comment (ChernikovVolodymyr) [reply app/admin/users.rb]: Done
Addressed here https://github.com/vova/war_shelter/pull/68/commits/4e4c286a607a201f83ff67501e343734d2ebb10e

File: app/admin/users.rb (hunk 2 above)
Comment (ChernikovVolodymyr) [reply app/admin/users.rb]: Done
Addressed here https://github.com/vova/war_shelter/pull/68/commits/4e4c286a607a201f83ff67501e343734d2ebb10e

**Pull request merged**